snp_motion_planning/src/planning_server.cpp

```diff
@@ -401,6 +418,9 @@ class PlanningServer
       // Get the default minimum distance allowable between any two links
       auto min_contact_dist = get<double>(node_, MIN_CONTACT_DIST_PARAM);
 
+      auto cart_tolerance_vector = get<std::vector<double>>(node_, TRAJOPT_CARTESIAN_TOLERANCE_PARAM);
```

Comment:
We probably need to check the size of this vector to make sure it's size 6. Seems easy to pass the wrong number of values on accident which could result in some unexpected behavior

Reply:
Added an exception if it is the wrong size.